src/utils/chain.ts

```diff
@@ -15,18 +15,43 @@
 import cli from 'cli-ux';
 import { getClient } from '../client';
 
+let blockID: string;
```

Comment:
Please rename the variable to be more intuitive.
```suggestion
let blockIDAtSnapshotHeight: string;
```